README.md

```diff
@@ -47,6 +47,8 @@ In the case when your request fails because the access token has expired, the id
 
 Thanks to the identity proxy, you won't have to worry about implementing this retry logic yourself. Just make the calls to your APIs and let the proxy handle the tokens. For more details on how the identity proxy accomplishes this, review this article: [Service Workers as an Identity Proxy](./service_workers.md).
 
+AppAuthHelper has two ways to renew an expired access token: a silent authorization code grant (the default behavior), or a refresh token grant. If this happens while the user still has a valid session within the OP (and the OP no longer prompts for consent), a new access token can be silently obtained by initiating a new authorization code grant within a hidden iframe. If you do not want your RP to depend on an active session within the OP (or if the OP requires consent for each authorization code grant) then you can tell AppAuthHelper to use a refresh token grant instead. Be sure the OP has been configured to allow this RP to use the refresh token grant if you choose this option.
```

Comment:
I know this library is generic (that is, not specific to ForgeRock environment and works with other OPs). So, I don't know whether/where it is worth mentioning that (currently) the refresh token is generated by AM before the access token modification script runs, which is less obvious fact than that the session info is not available during the refresh. Could be a factor for choosing renewal method that the claims may not persist by using a refresh token: https://bugster.forgerock.org/jira/browse/OPENAM-15340